app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -137,6 +137,12 @@ class Settings private constructor(
         default = true
     )
 
+    // If any of the prefs have been modified, quit displaying the fenix moved tip
+    fun shouldDisplayFenixMovingTip(): Boolean =
+        preferences.getBoolean(appContext.getString(R.string.pref_key_migrating_from_fenix_nightly_tip), true) &&
```

Comment:
should this be || instead of &&?

Reply:
I don't think so. Each of these default to "true" and each user will get served *one* of these fenix moving tips. If *any* of them become "false" we no longer want to display the tip.

Reply:
If it was an OR here, then they'd always get the tip and have no way to set the other pref keys to false :)